src/states/game/game.gd

```diff
@@ -273,16 +282,24 @@ func draw_card() -> void:
 		if deck.size() == 0:
 			return
 
+	for card in card_nodes.get_children() as Array[DualCard]:
+		card.draggable = false
+
 	var dual_card_data = deck.pop_front()
 	hand.append(dual_card_data)
-	var card = preload("res://src/cards/dual_card.tscn").instantiate()
-	card_nodes.add_child(card)
-	card.initialize(dual_card_data)
-	card.dropped_card.connect(self._on_card_dropped)
+	var new_card = preload("res://src/cards/dual_card.tscn").instantiate()
```

Comment:
`:=` 😈

Reply:
i will now commit die

Reply:
smh i keep getting blamed for other people's code 😤